quickstart/sample-apps/k3s-clustered/Makefile

```diff
@@ -0,0 +1,46 @@
+include ../Makefile
+VM_NAME := <INTEGRATION_SLUG>-sample-app
+VM_CLUSTER_MAIN := $(VM_NAME)-k3s-main
+CONFIG_FILE_DIR := jinja/variables
+CONFIG_FILE_VARS := $(CONFIG_FILE_DIR)/monitoring-config.yaml
+CONFIG_OUT_FILE := monitoring-config.yaml
+TEMPLATE_FILE_DIR := jinja/templates
+TEMPLATE_FILE := $(TEMPLATE_FILE_DIR)/monitoring-config-template.yaml
+RIVER_FILE := configs/suo.river
```

Comment:
do we need to call it river? I think this term is no longer in use, as river language is renamed to alloy with .alloy file suffix

Reply:
Ah thank you, I thought it sounded a bit outdated, will do a pass